src/Control/Comonad/Traced/Class.purs

```diff
@@ -47,3 +51,15 @@ censor f (TracedT tr) = TracedT ((f >>> _) <$> tr)
 
 instance comonadTracedTracedT :: (Comonad w, Monoid t) => ComonadTraced t (TracedT t w) where
   track t (TracedT tr) = extract tr t
+
+lowerTrack :: forall t m w a. ComonadTrans t => ComonadTraced m w => m -> t w a -> a
+lowerTrack m = track m <<< lower
```

Comment:
is this an implementation detail, or something to be exported? (It's currently being implicitly exported).

Reply:
whoops, yeah this should not be exported